sql-plugin/src/main/spark320/scala/com/nvidia/spark/rapids/shims/GetSequenceSize.scala

```diff
@@ -39,10 +39,7 @@ package com.nvidia.spark.rapids.shims
 import ai.rapids.cudf._
 import com.nvidia.spark.rapids.Arm._
 
-import org.apache.spark.unsafe.array.ByteArrayMethods.MAX_ROUNDED_ARRAY_LENGTH
```

Comment:
We'll need to add `{"spark": "350db"}` to support Databricks 14.3.

Reply:
I'm closing this out.  As discussed, we'll chase down Databricks 14.3's changes in a follow-up.